pkg/file/file.go

```diff
@@ -0,0 +1,39 @@
+package file
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/google/go-github/v28/github"
+)
+
+var (
+	// ErrNotFound is the error returned by GetFile when the target file does not exist.
+	ErrNotFound = errors.New("file not found")
+)
+
+// GetFile returns a handle to a file in a GitHub repository by branch and path.
+// If an empty value is provided as the name of the branch, the file is looked up in the default one.
+func GetFile(ctx context.Context, c *github.Client, repositoryOwner, repositoryName, branch, path string) (*github.RepositoryContent, error) {
+	var (
+		o *github.RepositoryContentGetOptions
+	)
```

Comment:
No need for brackets in this case.

Reply:
I know, but I prefer to use them as I believe it makes things more readable. In any case, I've removed them.